frontend/integration-tests/tests/metalkube/dashboard.scenario.ts

```diff
@@ -0,0 +1,70 @@
+import { browser } from 'protractor';
+const execSync = require('child_process').execSync;
+
+import { appHost } from '../../protractor.conf';
+import * as dashboardView from '../../views/metalkube/dashboards.view';
+
+describe('Inventory card', () => {
+  it('Node count is displayed', async() => {
+    await browser.get(`${appHost}/dashboards`);
+    await dashboardView.isLoaded();
+    browser.sleep(10000);
```

Comment:
why do we have to wait 10s here?

Reply:
The counters on the dashboard only start to populate several seconds after the page is already loaded. And it takes them quite a few more seconds to fully populate... I couldn't find any other way to see when the page is really ready.

Reply:
we could optimize a bit. What about waiting for the first line to appear and then wait 2-3 sec?